_includes/sidebar.html

```diff
@@ -38,7 +38,7 @@
     <p>&copy; {{ site.time | date: '%Y' }}. All rights reserved.</p>
 <p>
   {% for tongue in site.languages %}
-<a {% if tongue == site.active_lang %}style="font-weight: bold;"{% endif %}" href="{% if tongue == site.default_lang %} {{site.baseurl}}{{page.url}} {% else %} {{site.baseurl}}/{{ tongue }}{{page.url}} {% endif %}">{{ tongue }}</a> |
+<a {% if tongue == site.active_lang %}style="font-weight: bold;"{% endif %} href="{% if tongue == site.default_lang %} {{site.baseurl}}{{page.url}} {% else %} {{site.baseurl}}/{{ tongue }}{{page.url}} {% endif %}">{{ tongue }}</a> |
```

Comment:
I think so. I countered it while copying the code.
View sourcing section " en | es | de | fr | " in the website gives the mentioned output.